bin/travis/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+source tools.sh
+
+bloopDir="bloop-server"
+[ -d "$bloopDir" ] && rm -rf "$bloopDir"
+mkdir "$bloopDir"
+
+bloopVersion=$(get_bloop_version)
+bloopURL="https://github.com/scalacenter/bloop/releases/download/v$bloopVersion/install.py"
+
+wget "$bloopURL"
```

Comment:
no need to call python if wget fails
```suggestion
wget "$bloopURL" && python install.py --dest $bloopDir
```